src/Sulu/Bundle/AdminBundle/Resources/js/components/Snackbar/tests/Snackbar.test.js

```diff
@@ -1,50 +1,56 @@
 // @flow
 import React from 'react';
-import {mount, shallow} from 'enzyme';
+import {fireEvent, render, screen} from '@testing-library/react';
 import Snackbar from '../Snackbar';
 
 jest.mock('../../../utils/Translator', () => ({
     translate: jest.fn((key) => key),
 }));
 
 test('Render an error snackbar', () => {
-    const snackbar = mount(<Snackbar message="Something went wrong" onCloseClick={jest.fn()} type="error" />);
-    expect(snackbar.render()).toMatchSnapshot();
+    const {container} = render(<Snackbar message="Something went wrong" onCloseClick={jest.fn()} type="error" />);
+    expect(container).toMatchSnapshot();
 });
 
 test('Render an updated error snackbar', () => {
-    const snackbar = mount(<Snackbar message="Something went wrong" onCloseClick={jest.fn()} type="error" />);
-    snackbar.setProps({message: 'Something went wrong again'});
-    expect(snackbar.render()).toMatchSnapshot();
+    const {container, rerender} = render(<Snackbar
+        message="Something went wrong"
+        onCloseClick={jest.fn()}
+        type="error"
+    />);
+    rerender(<Snackbar message="Something went wrong again" onCloseClick={jest.fn()} type="error" />);
+    expect(container).toMatchSnapshot();
```

Comment:
Oh interesting hever used `rerender` yet but seems to correct way to test props update: https://testing-library.com/docs/react-testing-library/api/#rerender 👍